src/components/Firebase/firebase.js

```diff
@@ -64,6 +67,13 @@ default:
 class Firebase {
   constructor() {
     app.initializeApp(config);
+    this.auth = app.auth();
   }
+
+
+  testLogin = (email, password) => {
```

Comment:
Function name is not very descriptive as it is not for testing purposes but actual login